### ISSUE TITLE
TagInput - preset v-model OptionsPropItem values aren't displayed properly

### ISSUE THREAD

**Issue by joeyberkovitz** [oruga-ui/oruga#1148]
### Overview of the problem

**Oruga** version: 0.9.2

### Description

If using a taginput with OptionsPropItem entries, the values are handled properly when added via autocomplete. If preset via v-model upon the taginput creation with an OptionsPropItem object provided to the v-model, it tries to display the entire object instead of just the OptionsPropItem label

**Comment by mlmoravek** [comment]
Hey, the value of the `v-model` is only the value declared on the `value` attribute of an `OptionsPropItem` object, not the whole object. So the `OptionsPropItem` is a wrapper of the real `value` of an option and the associated `label` (and some other attributes if needed).

A more detailed description of the `options` prop and its declaration types and behaviour will surely follow in the docs!

**Comment by mlmoravek** [comment]
See #1126

**Comment by joeyberkovitz** [comment]
Yeah, I think I was running into an issue where if an object is initially passed into a taginput with no options populated (since they autocomplete), it just displays the entire object as JSON.

If you alternatively populate the options via autocomplete, then select an option, the value displayed is the label from the selected option's `OptionsPropItem`. So it might be convenient for the TagInput model to either be, or accept an `OptionsPropItem`, otherwise it's just a bit odd - although it does kind of make sense - that the model values all must exist in the provided `options` list to be displayed properly.
